Matchismo/Matchismo/Model/SetCard.mm

```diff
@@ -0,0 +1,165 @@
+//
+//  SetCard.m
+//  Matchismo
+//
+//  Created by Otto Olkkonen on 15/06/2022.
+//
+
+#include "SetCard.h"
+
+@interface SetCard()
+- (BOOL)allSameNoOfShapes:(NSArray<SetCard *> *)otherCards;
```

Comment:
Again here - better not to use the "no" abbreviation.  A bit of verbosity is fine if it helps readability.

Reply:
Though I'd add that these method names don't quite make sense. Or at least - they're not clear.
What exactly are "allSameNumberOfShapes" and "allDifferentNumberOfShapes" checking for?